tests/virus_total_agent_test.py

```diff
@@ -334,3 +335,61 @@ def testVirusTotalAgent_whenFileHasNoPath_shouldReportWithHash(
         "Analysis of the target `44d88612fea8a8f36de82e1278abb02f`:\n|Package|  Result  |  \n"
         "|-------|----------|  \n|Bkav   |_Safe_    |  \n|Elastic|_Malicous_|  \n"
     )
+
+
+def testPrepareTargets_whenIPv4AssetReachCIDRLimit_raiseValueError(
```

Comment:
The unit test, should reflect the actual method being tested, not the implied one.
Apply to the rest please!